FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/Models/FhirToDataLakeSplitSubJobCandidates.cs

```diff
@@ -0,0 +1,46 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Health.AnalyticsConnector.Core.Jobs.Models
+{
+    public class FhirToDataLakeSplitSubJobCandidates
```

Comment:
seems `FhirToDataLakeSplitSubJobCandidates` is not a model class? how about moving out from models?

Reply:
We can consider candidates as a model with related functions.